src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/MismatchStrategy.java

```diff
@@ -41,8 +41,8 @@ public enum MismatchStrategy {
 
     /**
      * In a case if matching allow/disallow rule was not found,
-     * import will be disallowed by default.
+     * import will be forbidden by default.
      */
-    DISALLOWED,
```

Comment:
dont we need both ?

Reply:
we need both, but we can put comment that it is synonym.